README.md

```diff
@@ -0,0 +1,72 @@
+# bookList-website
+
+This is a project to help you find and remember the best books you read.
+You can add favourite books, or books you'd like to read, you can create a book collection and this information will be stored and saved for you even if the browser is closed, you can access your collection anytime anywhere, a must have app with easy UI/UX.
+
+# Project Name : Awesome Books ♻️
+
+![screenshot}](./assets/Booklist2.png)
+
```

Comment:
[OPTIONAL] The link to your screenshot is not working properly. Kindly look into it to make sure that the image is displayed as expected.